tensorlayer/models/seq2seq.py

```diff
@@ -0,0 +1,154 @@
+#! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+import tensorflow as tf
+import tensorlayer as tl
+import numpy as np
+from tensorlayer.models import Model
+from tensorlayer.layers import Dense, Dropout, Input
+from tensorlayer.layers.core import Layer
+
+
+class Seq2seq(Model):
+
+    """vanilla stacked layer Seq2Seq model.
+
+    Parameters
+    ----------
+    decoder_seq_length: int
+        The length of your target sequence
+    cell_enc : str, tf.function
+        The RNN function cell for your encoder stack, i.e. tf.keras.layers.GRUCell
```

Comment:
'i.e.' means 'in other words'. Please use 'e.g.' (means 'for example') here and other places accordingly.